### ISSUE TITLE
Add `input` to option bag for `spawn` & `spawnSync`

### ISSUE THREAD

**Issue by crowlKats** [denoland/deno#14297]
Needs #11618 first.
Ref https://github.com/denoland/deno/pull/11618#discussion_r849432157

**Comment by nayeemrmn** [comment]
From https://github.com/denoland/deno/pull/14358#issuecomment-1106621570, I think we should have:
```ts
export interface SpawnOptions {
  // ...
  stdin?: "piped" | "inherit" | "null" | Uint8Array;
}
```
I believe it should apply to `spawnChild()` as well.

**Comment by superboss224** [comment]
Yes, we should add it to execute commands with big input (like md5), but which have a short and simple result

**Comment by iuioiua** [comment]
#11618 has now been merged. This would be a great little feature to have!
